src/main/kotlin/com/ajaxproject/warehouse/service/WaybillServiceImpl.kt

```diff
@@ -43,6 +79,13 @@ class WaybillServiceImpl(
         )
     }
 
+    fun WaybillCreateDto.mapToEntity(): Waybill = Waybill(
+        id = null,
+        customer = customerRepository.findById(customerId as Int)
+                .orElseThrow { NotFoundException("Customer with id $customerId not found") },
+        date = date as LocalDate    // TODO ask if this is OK
```

Comment:
```suggestion
        date = date!!
```
if you 100% sure then ok